[approved] feat: add Executive Council page to handbook (#113)
pyOpenSci/handbook: governance/executive-council.md

> (pyos-executive-council)=

Author: Add a section - that says voted on and has a date... between operations and governance part. say revised on....

Author: done!

  including updates to the Code of Conduct.
- **Significant budget deviations**: Decisions involving major deviations from the
  approved annual budget.
- **Financial health implications**: Any decision with substantial implications for
  pyOpenSci’s financial stability.
- **Strategic direction**: Approval of major changes to organizational strategy and
  development roadmaps.
- **Executive director’s salary**: The EC independently determines the salary of the
  Executive Director.

:::note
The Executive Director is authorized to make all other decisions. However, the
Executive Council serves as a resource for the Executive Director and may provide
feedback and input upon request.
:::

### Decision-making process

The topics above require a motion and a vote for approval. Motions and approval can be made in Executive Council meetings or asynchronously via email or Slack. Motions and votes will be noted in the EC meeting minutes.

:::{important}
All decisions made by the Executive Council must align with the legal
obligations and policies of pyOpenSci’s fiscal sponsor—[Community Initiatives](https://communityin.org/).
:::

> ## Collaboration between the Executive Director and the Executive Council

Author: Move this section into the "operations at the bottom too" . 

and remove the staff lines from the document. (we can use may request feedback - in the other operations section).

Author: Moved this and lazy consensus to the operations section.

The Executive Director may request feedback

> from the Executive Council on significant or public-facing decisions; these types of decisions do not require a formal vote. These discussions should

Author: Remove and put in operations "These discussiosn should occur in the .... "

> occur in the **executive-council** Slack channel to ensure transparency and

Author: ```suggestion
occur in the **executive-council** Slack channel to ensure 
```

Author: entire section moved to operations.

collaborative input. Examples include:

* Significant public-facing changes to the pyOpenSci website.

> * Decisions impacting staff or human resources (e.g., hiring or termination).

Author: ```suggestion
```

Author: removed locally

* Operational policies for staff (unless they carry major financial implications).
* Strategic decisions about pursuing proposals or writing grants.
* Substantial changes to pyOpenSci programs.
* Strategic decisions about pursuing proposals or writing grants.
* Substantial changes to pyOpenSci programs.

By maintaining open communication, the Executive Council and Executive Director ensure that pyOpenSci’s
decisions align with its mission and strategic goals.

> ### Lazy consensus for decisions that don't require a vote

Author: This  section could go in how the ex council meets and communicates section.

Author: moved

Lazy consensus is applied for decisions such as those listed above that do not require a formal vote.

Lazy consensus means that decisions are approved by default unless a member of
the Executive Council explicitly raises an objection within a predefined period.
This allows decisions to move forward efficiently while providing an
opportunity for discussion.

When objections are raised, further discussion occurs. This ensures decisions reflect the collective agreement of the Council
while minimizing unnecessary delays.

> ## How the Executive Council meets and communicates

Author: And then after adding lazy consensus move this to an operations  section of the guide - maybe at the bottom of this page --

add a callout - everything below is just operations and can be changed as need be. everything above is voted on and requires a vote to change

Author: moved

# pyOpenSci Executive Council

The pyOpenSci Executive Council (EC) has four primary functions:

1. **Guides the Organization**: The Executive Council defines and directs pyOpenSci’s high-level [mission, vision, and values](mission-values) and sets its strategic direction.

2. **Advocates for pyOpenSci**: The Executive Council serves as an advocate and ambassador for the organization, leveraging members’ networks to enhance its reputation and support fundraising efforts.

3. **Provides Strategic Insight**: The Executive Council offers perspective and guidance on major strategic decisions to ensure the organization’s long-term success.

4. **Holds the [Executive Director](executive-director) accountable**: The Executive Council holds the Executive Director accountable while also providing high-level strategic and financial guidance and oversight.

The Executive Director is currently a voting member of the Executive Council for items unrelated to their salary and accountability.

> The Executive Council holds broad expertise in building and running a scientific / data science nonprofit organization. It is complemented by the [Advisory Council](advisory-council), which offers domain-specific expertise, guidance, and input.

Reviewer: do we even need 'scientific / data science' here? I could imagine very good EC members from other domains

Reviewer 2: I agree, I don't think we should include that.

## How the Executive Council works

The pyOpenSci Executive Council operates through open discussions, collaborative
decision-making, and a commitment to transparency. By leveraging diverse
perspectives, the Council ensures that its guidance aligns with the organization’s
[mission, vision, and values](mission-values) while meeting the needs of the
growing pyOpenSci community.

The Council employs flexible and adaptive practices, such as the **lazy consensus**

> approach, to ensure decisions are made efficiently while fostering thoughtful

Reviewer 2: 'Lazy consensus' is a tool that we do use around certain types of decisions. But I think having it in this high level section suggests it's our standard approach, which is not true for things like, e.g. budget. So, I would recommend we move this somewhere else, and leave this sentence just as 

'...adaptive practices to ensure decisions are made..."

Author: ok removed that part. and we can include it below.

dialogue and collective alignment.

## Executive Council Roles & Responsibilities

> Each Executive Council (EC) member dedicates no more than **four to six hours per month** to pyOpenSci activities, including meeting attendance. Executive Council members are expected to fulfill the following responsibilities:

Reviewer: I think 4-6 hours on top of meeting attendance (esp if this is phrased as a hard upper bound)

Author: @kcranston so you think 4-6 hours + 1.5 hours a month for a meeting? You would know the time commitment better than I would at this point!! We could also write this as it dedicates approximately 4-6 hours a month. I just want to correctly represent the actual time!

Reviewer: Hmmm... you are probably right that the 4-6 hour estimate includes meeting time. Let's leave this as is.

Author: Perfect. No changes here then. Good to discuss and have a better sense of the actual time involved.

- Attend at least 80% of monthly Executive Council meetings.
- Participate in asynchronous commitments, which include:
  - Suggesting agenda items for meetings.
  - Communicating with the Executive Director (ED) via Slack.
  - Providing timely reviews of materials and documents.
  - Connecting the Executive Director with potential funding opportunities.
- Evaluate the performance of the Executive Director on a yearly basis.
- Approve expense reports.

> - Take meeting minutes and ensure they are [published in this Handbook](ec-minutes).

Reviewer 2: change to published in the [Executive Council Minutes section of the Handbook](link)

Author: More specific. I made this change locally.

The Executive Council is currently led by the Executive Council Chair, who approves the Executive Director’s vacation time off requests and sick leave.

:::{note}
While travel is not explicitly required, members may have opportunities to attend meetings on behalf of pyOpenSci. Travel may be supported by pyOpenSci if needed.
:::

## How the Executive Council makes decisions

The Executive Council supports the Executive Director by serving as a resource
for input when needed. However, certain items require explicit Executive Council
approval.

### Decision-making process

> The Executive Council generally follows a **lazy consensus** approach for decision-making:

Reviewer: this is true, except for the cases listed below that require a vote

Reviewer 2: I think we can flip these sections, starting with 'Decisions requiring a vote' and then go into how decisions are made. 

So

## How the Executive Council makes decisions

The Executive Council is required to review and approve decisions affecting the fiduciary aspects of the organization, this includes mission/vision/values, governance policies, budget, Executive Director compensation, and new significant strategic initiatives. The Executive Council also supports the Executive Director by serving as a resource for input when needed. 

All decisions made by the Executive Council must align with the legal
obligations and policies of pyOpenSci’s fiscal sponsor—[Community Initiatives](https://communityin.org/).
This ensures compliance with overarching governance structures and financial
responsibilities.

### Decisions requiring an Executive Council vote

what we already have in this section

### Decision making process

The topics above require a motion and a vote for approval. Motions and approval can be made in Executive Council meetings or asynchronously via email or Slack. Motions and votes will be noted in the EC meeting minutes.  

## Collaboration with the Executive Director

...
- Strategic decisions about pursuing proposals or writing grants.
- Substantial changes to pyOpenSci programs.

In these cases, *lazy consensus* can be employed. About lazy consensus (noted below where to add that section)

  pyOpenSci’s financial stability.
- **Strategic direction**: Approval of major changes to organizational strategy and
  development roadmaps.
- **Executive director’s salary**: The EC independently determines the salary of the
  Executive Director.

:::note
The Executive Director is authorized to make all other decisions. However, the
Executive Council serves as a resource for the Executive Director and may provide
feedback and input upon request.
:::

## Collaboration between the Executive Director and the Executive Council

While not requiring a formal vote, the Executive Director may request feedback
from the Executive Council on significant or public-facing decisions. These discussions should
occur in the **executive-council** Slack channel to ensure transparency and
collaborative input. Examples include:

- Significant public-facing changes to the pyOpenSci website.
- Decisions impacting staff or human resources (e.g., hiring or termination).
- Operational policies for staff (unless they carry major financial implications).
- Strategic decisions about pursuing proposals or writing grants.
- Substantial changes to pyOpenSci programs.

> 

Reviewer 2: Add here about lazy consensus approach

Author: great - i've reorganized the section based on your suggestions @tracykteal i just pushed up the changes. 🚀

- Executive Council members are expected to serve for 2–3 years.
- At the end of each 2-year term, members may choose to step down or renew their
  commitment based on mutual agreement with the Council.

As needed, the Executive Council may conduct periodic evaluations to ensure its composition continues
to meet the organization’s needs.

### Decision-making process

The topics above require a motion and a vote for approval. Motions and votes can be made in Executive Council meetings or via email, Slack of GitHub. Motions and votes will be noted in the Executive Council meeting minutes.

:::{admonition} Executive Council Approval
:class: important

> The above governance for the Executive Council was last updated, voted on, and approved on **insert-date-here**. All content below this section can be modified as needed and does not require a vote.

Author: ```suggestion
The above governance for the Executive Council was last updated, voted on, and approved on **23 January 2025**. All content below this section can be modified as needed and does not require a vote.
```

**Executive Director--Executive Council Chair meetings:** The Executive Director also meets regularly with the pyOpenSci Board Chair to ensure alignment on key issues and strategic direction.

## EC communication platforms & tools

### Slack: primary communication platform

> The **[Executive Council Slack channel](pyos-slack)** is the primary tool for communication between monthly meetings. All updates, discussions, and requests for input happen here; email is not used for EC communications.

Reviewer 2: ...happen here; email is rarely used for EC communications.

* smooth transitions,
* that new members are set up for success and
* that departing members are appropriately transitioned out of their roles.

### Onboarding a new Executive Council member

When a new member joins the Executive Council, the following steps should be
completed to integrate them into the team:

1. **Access to tools and resources**:
   - Add the new member to the pyOpenSci Executive Council shared [Google Drive](google-drive).
     This drive is where we share meeting notes,
     agendas, budgets and other resources.
   - Add the new member to the Executive Council Slack channel, the primary
     communication platform for the Council.

2. **Orientation**:
   - Provide a link to this Handbook, which outlines the Council’s
     responsibilities, decision-making processes, and operational policies.
   - Schedule an initial orientation meeting, if needed, to familiarize the new member
     with pyOpenSci’s mission, vision, values, and ongoing initiatives.

3. **Notifications**:
   - Notify the pyOpenSci's Fiscal Sponsor (Community Initiatives) of any changes in roles
     that impact the chair position or budget oversight responsibilities.

> 4. **Welcome and integration**:

Reviewer: to the council, and to the pyOS community?

Author: Good point -- i've clarified this. 

4. **Welcome and integration**:
   - Introduce the new member to the Executive Council during a meeting and to the pyOpenSci community through Slack
     announcement. Highlight their expertise and how it complements the team’s
     needs.

- At the end of each 2-year term, members may choose to step down or renew their
  commitment based on mutual agreement with the Council.

As needed, the Executive Council may conduct periodic evaluations to ensure its composition continues
to meet the organization’s needs.

### Decision-making process

The topics above require a motion and a vote for approval. Motions and votes can be made in Executive Council meetings or via email, Slack of GitHub. Motions and votes will be noted in the Executive Council meeting minutes.

:::{admonition} Executive Council Approval
:class: important

The above governance for the Executive Council was last updated, voted on, and approved on **insert-date-here**. All content below this section can be modified as needed and does not require a vote.
:::

## Executive Council operations

### Collaboration between the Executive Director and the Executive Council

The Executive Director may request feedback
from the Executive Council on significant or public-facing decisions; these decisions do not require a formal vote. These discussions generally
occur in the **executive-council** Slack channel to ensure
collaborative input. Examples include:

> - Significant public-facing changes to the pyOpenSci website.
> - Operational policies for staff (unless they carry significant financial implications).
> - Strategic decisions about pursuing proposals or writing grants.
> - Substantial changes to pyOpenSci programs.
> - Strategic decisions about pursuing proposals or writing grants.
> - Substantial changes to pyOpenSci programs.

Author: ```suggestion
- Significant public-facing changes to the pyOpenSci website.
- Operational policies for staff (unless they carry significant financial implications).
- Strategic decisions about pursuing proposals or writing grants.
- Substantial changes to pyOpenSci programs.
```

Author: Good catch @kcranston that must have happened when i moved things around!! i totally missed it.

Generally, items with legal or fiduciary implications require a formal vote,
with the outcome documented in this handbook's Executive Council [meeting minutes](ec-minutes) section.

The EC will vote on the following:

- **Annual budget**: Approval of the organization's yearly budget.
- **Mission, vision, and values**: Decisions related to pyOpenSci’s core principles.
- **Governance and policy changes**: Approval of changes to governance structures,
  including updates to the Code of Conduct.
- **Significant budget deviations**: Decisions involving major deviations from the
  approved annual budget.
- **Financial health implications**: Any decision with substantial implications for
  pyOpenSci’s financial stability.
- **Strategic direction**: Approval of major changes to organizational strategy and
  development roadmaps.
- **Executive director’s salary**: The EC independently determines the salary of the
  Executive Director.

:::note
The Executive Director is authorized to make all other decisions. However, the
Executive Council serves as a resource for the Executive Director and may provide
feedback and input upon request.
:::

### Decision-making process

> The topics above require a motion and a vote for approval. Motions and votes can be made in Executive Council meetings or via email or Slack. Motions and votes will be noted in the Executive Council meeting minutes.

Reviewer 2: ... via email, Slack or GitHub.

Author: fixed.

#### Slack: primary communication platform

The **[Executive Council Slack channel](pyos-slack)** is the primary tool for communication between monthly meetings. All updates, discussions, and requests for input happen here; email is rarely used for EC communications.
EC members are expected to check Slack semi-regularly to stay informed.

- **Feedback requests:** It is understood that EC members have jobs and lives outside of their EC responsibilities. As such, when the Executive Director (ED) requests input on Slack, they will specify what feedback is needed and provide clear response timelines and areas to focus on.
- **Transparency:** The Slack channel is also used to share updates about changes that occur outside of formal monthly meetings. Examples of such updates include new hires, event updates, and celebrating successes.

#### Google Drive: document sharing and collaboration

The pyOpenSci Executive Council uses a shared **[Google Drive](google-drive)** within the pyOpenSci Google Workspace to store and share documents. This includes meeting notes, agendas, and other important materials. Members are encouraged to access and update shared resources using the shared Drive.

> ### Selection of Executive Council members

Reviewer 2: This is a part of the governance structure, so we can move this section up to the top section under 'Decision-making process'

Author: I moved it up. I'm assuming we'd collectively vote on new members so it's above the voting section.

2. **Identify Candidates**:
   - Potential candidates are identified based on their experience, expertise,
     and alignment with pyOpenSci’s mission, vision, and values. This may include
     leaders in the open science community, individuals with financial or
     governance expertise, or others with relevant nonprofit governance skills.

3. **Review and nominate the new member**:
   - Candidates are discussed and nominated by current Council members during a
     meeting or through the Executive Council Slack channel. After nomination, the Executive Director
     will contact the candidate to see if they are interested/available.

5. **Onboard the new candidate**:
   - Once approved, new members are onboarded following the onboarding process
     outlined in this document.

#### Terms and evaluation

- Executive Council members are expected to serve for 2–3 years.
- At the end of each 2-year term, members may choose to step down or renew their
  commitment based on mutual agreement with the Council.

As needed, the Executive Council may conduct periodic evaluations to ensure its composition continues
to meet the organization’s needs.

> ### Onboarding and offboarding Executive Council members

Author: ok this has been moved.
